langlib/lang.map/src/main/ballerina/src/lang.map/Module.md

```diff
@@ -1,3 +1,5 @@
 ## Module Overview
 
-This module provides lang library map operations defined by the language specification 2019R2.
+This module provides lang library map operations defined by the language specification 2019R3. All the functions listed
+below are supported by the `map` type. Records do not support any functions which violate their respective inherent 
```

Comment:
```suggestion
in this module are supported by mapping types. Records do not support any functions that violate their inherent 
```

Reply:
Can’t say mapping types since records only support a subset of the functions..

Reply:
Can’t say mapping types since records only support a subset of the functions..

Reply:
But saying `map` types might also be misleading, right? It's like they are only supported with `map`-typed variables. Then we talk about records, and there's a disconnect.

How about,
```suggestion
in this module are supported by mapping types, as long as the function does not violate the inherent type.
```

Functions like remove can also fail on maps, if immutable.